packages/varnish/src/header/Header.tsx

```diff
@@ -53,7 +49,7 @@ const LogoGrid = styled.span`
   display: grid;
   grid-template-columns: repeat(4, auto);
   align-items: center;
-  gap: ${varnishLightTheme.spacing(1)};
+  gap: ${({ theme }) => theme.spacing(1)};
```

Comment:
just confirming that everything is still looking as expected on the docs? i didn't realize you didn't need to getTheme in order to reference theme here

Reply:
yes, the change says, use the current theme (rather than 'use the default theme')